### PR TITLE
feat: remove column password in table user

### PR DESCRIPTION
**Pull request by hoanhtnb**
## Description

 remove column password in table user
 user-service

## Ticket

https://trello.com/c/77WMDRjX/31-user-service-xo%C3%A1-c%E1%BB%99t-password-trong-db

## Type of change

- [ X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Build (new dependencies / configurations added)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore/Errand (add documentation/comments or fix spacing - no code was touched)
- [ ] Other:  

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have tested my code exhaustively
- [ ] I have updated corresponding changes to the documentation

### CODE REVIEW

File: apps/user-service/src/user/user.authen.service.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export class UserAuthenService {
     private userHelperService: UsersHelperService
   ) {}
 
-  async authenticateUserPassword(email: string, password: string) {
+  async authenticateUserEmail(email: string) {
```
Comment (dungnh1705) [inline apps/user-service/src/user/user.authen.service.ts:19]: please keep old logic.
this's out of scope for this ticket.

File: apps/user-service/src/user/user.controller.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ export class UserController {
   @Post("login")
   async login(@Body() payload) {
```
Comment (dungnh1705) [inline apps/user-service/src/user/user.controller.ts:16]: please keep old logic.
this's out of scope for this ticket.

File: apps/user-service/src/user/user.authen.service.ts (hunk 1 above)
Comment (dungnh1705) [reply apps/user-service/src/user/user.authen.service.ts:19]: @hoanhtnb

File: apps/user-service/src/user/user.controller.ts (hunk 2 above)
Comment (dungnh1705) [reply apps/user-service/src/user/user.controller.ts:16]: @hoanhtnb

**Review by nambach — COMMENTED** [review]
LGTM

**Review by nambach — APPROVED** [review]

**Review by dungnh1705 — APPROVED** [review]

**Pull request merged**